vipul/Gallery_System/about.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en-US">
+	<head>
+		<meta name="description" content="This is webpage for online photo gallery system.">
+		<meta name="keyword" content="online,photo,gallery.">
+		<title>Photo Gallery | About Us</title>
+		<link rel="stylesheet" href="style.css">
+	</head>
+	<body>
+		<div class="container">
+			<div class="header">
+				<h1>
+					<a href="index.html">Photo Gallery</a>
+				</h1>
+			</div>
+			<div class="list">
+				<ul class="nav">
+					<li><a href="index.html" class="active">HOME</a></li>
+					<li class="sublist">
+						<a href="wild.html">CATEGORY</a>
+						<ul class="dropdown">
+							<li><a href="wild.html">Wildlife</a></li>
+							<li><a href="landscape.html">Landscape</a></li>
+							<li><a href="portrait.html">Portrait</a></li>
+							<li><a href="fashion.html">Fashion</a></li>
+							<li><a href="macro.html">Macro</a></li>
+							<li><a href="bird.html">Bird</a></li>
+							<li><a href="nature.html">Nature</a></li>
+							<li><a href="food.html">Food</a></li>
+						</ul>
+					</li>
+					<li><a href="services.html">SERVICES</a></li>
+					<li><a href="about.html">ABOUT</a></li>
+					<li><a href="contact.html">CONTACT</a></li>
+				</ul>
+			</div>
+		</div>
+		<div class="main-content-about-us">
+			<p>About Us</p>
+			<img src="images/portrait5.jpg">
+			<div class="about-us-list">
+				<h3>
+					<p>	We specialize in wildlife photography, food photography, nature photography and portraits, often traveling to your destination to capture the perfect moment in the perfect place.
+						We will be there with you every step of the way to guarantee your special moments are captured for all time.
+						To us photography is about people being real and then letting us paint a picture of that moment to remember it forever.
+						This is the story that matters most: real people, real stories, real moments.
+						We are incredibly blessed to have a team of photographers who work with us on any given weekend.
+						Please take a few minutes to get to know us.
+						We would love to visit with you by phone or in person and talk about your event, portrait session.
+						Please reach out to us through the Contact Page.
+					</p>
+				</h3>
+			</div>
+		</div>
+		<div class="footer">
+			<p>
+				Copyright 2020 All rights reserved | This is developed by <a href="https://www.facebook.com/vipul.suryawanshi" target="_blank">Vipul Suryawanshi.</a>
+			</p>
+		</div>
+	</body>	
+</html>	
```

Comment:
remove all extra spaces
Add more content in footer
Add contact us in footer  
bad code indentation